File: public/phase-2-20160620022203/C/robot-s-task.c
// Name : Robot's Task
#include <stdio.h>
int main() {
	// input:
	// n : number of non negative integers
	// a : integers
	int n, a[1000], changes=1, index;
	scanf("%d", &n);
	for(index = 0; index< n; index++) {
		scanf("%d", &a[index]);
	}


	// write your code here
	// store your results in `changes`

	// output
	printf("%d", changes);
	return 0;
}

File: public/phase-2-20160620022203/C_SHARP/robot-s-task.cs
// Name : Robot's Task
using System;
namespace Application {
	class robot's_task {
		public static void Main(string[] args) {
		// input:
		// n : number of non negative integers
		// a : integers
		int index;
		int n = Convert.ToInt32(Console.ReadLine());
		int[] a = new int[n];
		string[] a_elements = (Console.ReadLine()).Split(' ');
		for(index=0;index<n;index++)
			a[index] = Convert.ToInt32(a_elements[index]);


		// write your code here
		// store your results in `changes`

		// output
		// Dummy Data
		int changes=1;

		Console.WriteLine(changes);

		}	
	}
}